7.0/Dockerfile

```diff
@@ -38,7 +38,9 @@ RUN yum install -y centos-release-scl wget && \
                   rh-php70-php-gmp" && \
     yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS --nogpgcheck && \
     rpm -V $INSTALL_PKGS && \
-    yum clean all -y
+    yum clean all -y && \
+    pecl install v8js-1.4.1 && \
+    echo extension=v8js.so > /etc/php/7.0/cli/conf.d/99-v8js.ini
```

Comment:
We should probably use the `PHP_VERSION` env var (see top of file) rather than hardcoded version in the path.

Are you sure `99-v8js` is the correct file name?

Reply:
Yes, your right. Shouldn't we use the `$PATH` as that is the correct path where is should store the file?
Something like `echo extension=v8js.so > $PATH/php`

Reply:
> Shouldn't we use the $PATH as that is the correct path where is should store the file?
I dont know. Where is the file supposed to be stored?

> Goal is we need to add the name inside php.ini.

Add what name? If that is the case, shouldnt you be editing this file too?